docs/pangeo-showcase.rst

```diff
@@ -85,6 +28,42 @@ Showcase Archive
    * - Date
      - Speaker
      - Title
+   * - 2023-05-17 12PM EDT
+     - Campbell Watson, IBM
+     - `The Geospatial Discovery Network: Accelerating geospatial science in the multicloud future <https://discourse.pangeo.io/t/wednesday-may-17th-searching-beyond-pixels-and-tiles/3558>`_ |Campbell DOI Badge|
+   * - 2023-05-10 4PM EDT
+     - Navid Constantinou, Australian National University
+     - `Open-source and reproducibility workflow within the Consortium for Ocean-Sea Ice Modelling in Australia (COSIMA) <Wednesday May 10th: Open-source and reproducibility workflow within the Consortium for Ocean-Sea Ice Modelling in Australia (COSIMA)>`_ |Constantinou DOI Badge|
+   * - 2023-04-26 4PM EDT
+     - Michael Barletta, University at Albany
+     - `Creating Interactive Plots to Display Weather Data Using the Holoviz/Geoviews Libraries in Python <https://discourse.pangeo.io/t/wednesday-april-26th-creating-interactive-plots-to-display-weather-data-using-the-holoviz-geoviews-libraries/3556>`_ |Barletta DOI Badge|
+   * - 2023-04-12 4PM EDT
+     - Abigail Bodner, NYU/ClimateMatch Academy
+     - `Climatematch Academy: using Pangeo tools to educate the world on climate <https://discourse.pangeo.io/t/wednesday-april-12th-climatematch-academy-from-data-to-action/3298>`_ 
```

Comment:
This line is missing a DOI Badge. Do we have a video recording/DOI for Abigail's presentation?